index.html

```diff
@@ -959,51 +980,11 @@ <h2>Status</h2>
         <h2>Presentations</h2>
 
         <p>
-Credentials MAY be composed by placing them into a data structure called a
-<a>presentation</a>. A <a>verifiable presentation</a> is a presentation that contains
-<a>verifiable credentials</a> and one or more proofs that are appropriate
-for the presentation. The basic structure of a <a>verifiable presentation</a>
-is provided below:
+Credentials MAY be used to make a <a>verifiable presentation</a>. The presentation
```

Comment:
The specification makes a distinction between presentations (which are not verifiable) and verifiable presentations (which are verifiable). The modified language goes back and forth between these definitions, which muddies the waters. What about this:

> Credentials MAY be used to create presentations, which are composed of information from one or more credentials. A verifiable presentation is a presentation that is composed of information from one or more verifiable credentials and is thus cryptographically verifiable. The information in a verifiable presentation may be directly or indirectly derived from one or more verifiable credentials. The basic structure of a verifiable presentation is provided below:

Reply:
Yes, I see how I muddied the distinction. I will revise, using your language as a guide.